lib/scenic/view.rb

```diff
@@ -14,11 +14,11 @@ def ==(other)
     end
 
     def to_schema
-      <<-DEFINITION.strip_heredoc
-        create_view :#{name}, sql_definition:<<-\SQL
-          #{definition}
-        SQL
+      <<-DEFINITION
 
+  create_view :#{name}, sql_definition: <<-\SQL
+    #{definition.indent(2)}
```

Comment:
That appears to work for some views but not others. Postgres does some weird formatting, evidently. I tried various configurations. The `indent(2)` there *does* have the desired effect.